review.css

```diff
@@ -0,0 +1,36 @@
+
+body {
+  font-size: 400;
+  text-align: center;
+  margin: 0;
+  display: block;
+}
+
+#main {
+  margin: 20px;
+  padding: 10px;
+  background: lightred;
```

Comment:
You should use hex code for colors for semantic reasons.

Reply:
Kindly consider phrasing your comments politely, as suggestions instead of orders or commands. 
For instance, instead of "You need to...." or "You should use hex code for colors for semantic reasons", 
consider using expressions such as "kindly consider using hex code for colors for semantic reasons"  or "please you can use hex code for colors for semantic reasons" etc.